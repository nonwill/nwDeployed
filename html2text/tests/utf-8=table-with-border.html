<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head><body style="word-wrap: break-word; -webkit-nbsp-mode: space; line-break: after-white-space;" class=""><span style="font-family: TimesNewRomanPSMT;" class="">Team,</span><div class="" style="font-family: TimesNewRomanPSMT;"><br class=""></div><div class=""><span style="font-family: TimesNewRomanPSMT;" class=""><span class="Apple-tab-span" style="white-space: pre;">	</span>We&nbsp;move to test ne release now. ðŸ˜€ In our first test phase, we do not alter our tests, but use the same tests against old. &nbsp;Our objective of this phase is to find new issues with OLD tests. &nbsp;You can see our progress from </span><font color="#0069d9" face="TimesNewRomanPSMT" class=""><span style="caret-color: rgb(0, 105, 217);" class=""><u class=""><a href="http://bla/summary04152020.html" class="">http://bla/summary04152020.html</a>.</u></span></font></div><div class=""><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;"><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;">The following table in the report shows our bug progress. &nbsp;You can see more information by clicking the links in the table.&nbsp;</div><table class=""><tbody class=""><tr class=""><td class=""><table id="bugTable" class="bugchart" style="height: 75px;" data-graph-container=".. .. .bugchart-container" data-graph-type="line" data-graph-row-for-series="1" data-graph-max-dataset-size="10" data-graph-stack-group="1" data-graph-yaxis-1-stacklabels-enabled="1" data-graph-yaxis-1-title-text="Number of Bugs" data-graph-datalabels-enabled="1" data-graph-title="System Testing Bugs" data-graph-legend-layout="vertical" data-graph-legend-align="right" width="37" border="1"><tbody class=""><tr class=""><td class=""><span style="font-family: arial, helvetica, sans-serif; font-size: 10pt;" class=""><strong class="">Date</strong></span></td>
<td class=""><span style="font-family: arial, helvetica, sans-serif;" class=""><span style="font-size: 13.3333px;" class=""><strong class="">4/15</strong></span></span></td>
<td class=""><span style="font-family: arial, helvetica, sans-serif;" class=""><span style="font-size: 13.3333px;" class=""><strong class="">4/8</strong></span></span></td>
</tr>
<tr class="">
<td class=""><span style="font-size: 10pt;" class=""><strong class=""><span style="font-family: arial,helvetica,sans-serif;" class="">New</span></strong></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://bugs" target="_bug" class="">69</a></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://bugs" target="_bug" class="">67</a></span></td>
</tr>
<tr class="">
<td nowrap="nowrap" class=""><span style="font-size: 10pt;" class=""><strong class=""><span style="font-family: arial,helvetica,sans-serif;" class="">Closed</span></strong></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://closed" target="_bug" class="">71</a></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;</span></td>
</tr>
<tr class="">
<td nowrap="nowrap" class=""><span style="font-size: 10pt;" class=""><strong class=""><span style="font-family: arial,helvetica,sans-serif;" class="">Deferred</span></strong></span></td>
<td class=""><span style="font-size: 10pt;" class=""><a href="http://deferred" target="_bug" class="">182</a></span></td>
<td class=""><span style="font-size: 10pt;" class=""><a href="http://deferred" target="_bug" class="">142</a></span></td>
</tr>
<tr class="">
<td class=""><span style="font-size: 10pt;" class=""><strong class=""><span style="font-family: arial,helvetica,sans-serif;" class="">HOT</span></strong></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://hot" class="">70</a></span></td>
<td class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://hot" class="">63</a></span></td>
</tr>
<tr data-graph-skip="1" class="">
<td class=""><span style="font-size: 10pt;" class=""><strong class=""><span style="font-family: arial,helvetica,sans-serif;" class="">odd(reported from old | rest)</span></strong></span></td>
<td style="text-align: center;" class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://odd" class="">1581</a>&nbsp;</span><br class=""><span style="font-size: 10pt;" class="">(<a href="http://odd" class="">1136</a>|<a href="http://odd" class="">389</a>)</span></td>
<td style="text-align: center;" class=""><span style="font-size: 10pt;" class="">&nbsp;<a href="http://odd" class="">1589</a>&nbsp;</span><br class=""><span style="font-size: 10pt;" class="">(<a href="http://odd" class="">1167</a>|<a href="http://odd" class="">422</a>)</span></td>
</tr>
</tbody>
</table>
</td>
<td class=""><span class="bugchart-container" style="width: 450px; height: 200px; font-size: 10pt;">&nbsp;</span></td></tr></tbody></table><div class=""><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;">&nbsp;<span class="Apple-tab-span" style="white-space:pre">	</span>Weâ€™d like to learn and understand new/enhanced features introducing. &nbsp;Please help us. &nbsp;:)</div><div class="" style="font-family: TimesNewRomanPSMT;"><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;"><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;">Thanks</div><div class="" style="font-family: TimesNewRomanPSMT;"><br class=""></div><div class="" style="font-family: TimesNewRomanPSMT;">A &amp; B</div></body></html>
