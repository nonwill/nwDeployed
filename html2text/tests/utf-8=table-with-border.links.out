Team,

WeÂ move to test ne release now. ðŸ˜€ In our first test phase, we do not alter our
tests, but use the same tests against old. Â Our objective of this phase is to
find new issues with OLD tests. Â You can see our progress from _h_t_t_p_:_/_/_b_l_a_/
_s_u_m_m_a_r_y_0_4_1_5_2_0_2_0_._h_t_m_l_[_1_]_.


The following table in the report shows our bug progress. Â You can see more
information by clicking the links in the table.Â 
 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
|_DD_aa_tt_ee_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_44_//_11_55_ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_44_//_88_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ |
|_NN_ee_ww_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_Â _6_9_[_2_]_ _ _ _ _ _ _ _ _ _ _ _ _|_Â _6_7_[_3_]_ _ _ _ _ _ _ _ _ _ _ _ |
|_CC_ll_oo_ss_ee_dd_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_Â _7_1_[_4_]_ _ _ _ _ _ _ _ _ _ _ _ _|_Â _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ | Â 
|_DD_ee_ff_ee_rr_rr_ee_dd_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_1_8_2_[_5_]_ _ _ _ _ _ _ _ _ _ _ _ _|_1_4_2_[_6_]_ _ _ _ _ _ _ _ _ _ _ _ |
|_HH_OO_TT_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_Â _7_0_[_7_]_ _ _ _ _ _ _ _ _ _ _ _ _|_Â _6_3_[_8_]_ _ _ _ _ _ _ _ _ _ _ _ |
|oodddd((rreeppoorrtteedd ffrroomm oolldd || rreesstt))|    Â _1_5_8_1[9]Â      |    Â _1_5_8_9[12]Â     |
|_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _|_(_1_1_3_6_[_1_0_]_|_3_8_9_[_1_1_]_)_|_(_1_1_6_7_[_1_3_]_|_4_2_2_[_1_4_]_)|

Â 
Weâ€™d like to learn and understand new/enhanced features introducing. Â Please
help us. Â :)


Thanks

A & B


############ RReeffeerreenncceess ############
   1. http://bla/summary04152020.html
   2. http://bugs
   3. http://bugs
   4. http://closed
   5. http://deferred
   6. http://deferred
   7. http://hot
   8. http://hot
   9. http://odd
  10. http://odd
  11. http://odd
  12. http://odd
  13. http://odd
  14. http://odd
